CHANGELOG.md

```diff
@@ -12,11 +12,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Node runtime extended with `ddc-staking` pallet
 
-## [2.34.0]
-### Changed
-- Updated the ss58 prefix to custom one from [ss58 registry](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json#L491)
-- New `spec_version` format. Mainnet runtime specification upgrade increments the second and third digits and resets two least significant digits now. Other upgrades should update these two least significant digits only.
```

Comment:
Can we keep the `spec_version` format description due to the runtime version increment required?

https://github.com/Cerebellum-Network/pos-network-node/pull/197/files#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aR132

Reply:
Will return it back to dev soon.